base/src/test/java/uk/ac/ebi/atlas/solr/analytics/query/AnalyticsQueryClientIT.java

```diff
@@ -44,127 +51,157 @@
     @Mock
     private RestTemplate restTemplate;
 
+    @Inject
+    private AnalyticsQueryClient goodSubject;
```

Comment:
Do we need the "good" subject and the (bad) subject?

Reply:
Whatever is being tested with the bad subject (and the `TestableAnalyticsQueryClient`) is a bit pointless. I’m pretty sure that anything worth testing can be achieved with the good subject only, but this was a tangent I didn’t want to take yet. It’s part of our technical debt list, though.